src/ips-nmos-api/src/http/subscription/add-token.js

```diff
@@ -0,0 +1,30 @@
+module.exports = () => {
```

Comment:
As you're using webpack ya can use the es6 import export system.


```
export const fetch ....
export const addAuthHeaders ....
export const addAuthQuery ....
```

Reply:
or `export function fetch () { ....`

Reply:
I've kept it the same as the other modules in that directory for consistency.. Think Luke uses the CommonJS style for the backend and the ES6 imports style for the app